rms/rms.go

```diff
@@ -44,9 +46,71 @@ func (r *RoomzMediaServer) disconnectHandler(s socketio.Conn, msg string) {
 }
 
 func (r *RoomzMediaServer) joinMediaRoomHandler(s socketio.Conn, data map[string]interface{}) {
+  log.Printf(":JoinMediaRoom: received data=%v", data)
+  userId, ok := data["user_id"].(int64)
+  if !ok {
+    log.Printf("invalid user_id")
+    return
+  }
+  roomId, ok := data["room_id"].(int64)
+  if !ok {
+    log.Printf("invalid room_id")
+    return
+  }
+  // TODO: Validate token.
+  room := r.roomMgr.GetRoom(roomId)
+  mediaMgr := room.GetMediaManager()
+  log.Printf("Starting broadcast for userId: %v", userId)
+  mediaMgr.StartBroastcast(userId)
+  room.Join(userId, s.ID())
+
+  log.Printf("Emitting \"NewMediaRoomyArrived\" to roomId: %v for userId: %v", roomId, userId)
```

Comment:
Random minor question: is `\"` the "Google" preffered way to use double quotes? Or would it be better to swap the whole line to use single quotes so that you don't need the backslash?

Reply:
Golang strings only are double quotes